swift/ios-shoppe-demo/ServiceManagers/FullStoryManager.swift

```diff
@@ -62,3 +62,7 @@ func fsLog(message: String, level: LogLevel = .info) {
 func fsModifyPrivacy(setting: PrivacySetting, of view: UIView) {
     FS.addClass(view, className: setting.rawValue)
 }
+
+func fsIdentify(id: String, userInfo: [String: Any]) {
+    FS.identify(id, userVars: userInfo)
```

Comment:
Is there a need to wrap this?  Callers of `fsIdentify` could seemingly also do `FS.identify(id, userVars: user.infoDict)` correct?

Reply:
nm; the manager's responsibility is to provide access to FS.